app/config/index.js

```diff
@@ -5,12 +5,16 @@ const storageConfig = require('./storage')
 
 // Define config schema
 const schema = Joi.object({
-  env: Joi.string().valid('development', 'test', 'production').default('development')
+  env: Joi.string().valid('development', 'test', 'production').default('development'),
+  ffcApiPath: Joi.string().required(),
```

Comment:
I'd maybe suggest renaming this from `ffcApiPath` as it's not really clear what it's used for.  It's used later as the get route appended to the query route, so what about `getStatementPath` or something?